lib/modules/manager/pipenv/artifacts.ts

```diff
@@ -24,38 +26,83 @@ import type {
 import { extractPackageFile } from './extract';
 import { PipfileLockSchema } from './schema';
 
-export function getPythonConstraint(
+export async function getPythonConstraint(
```

Comment:
```suggestion
async function getPythonConstraint(
```

not exyternally used, so no need to export

Reply:
I use this in runinstall